### PR TITLE
upgrade: `unbzip2-stream` to v1.0.11

### PR DESCRIPTION
**Pull request by jviotti**
We believe this issue fixes the "rawr i'm a dinosaur" error on certain
problematic images.

Gives this only happens on a minority of images, we couldn't confirm
that the fix really solves the issue, but we'll include the upgrade on
the next Etcher version and see how it goes in the real world.

Change-Type: patch
Changelog-Entry: Fix "rawr i'm a dinosaur" bzip2 error.
Fixes: https://github.com/resin-io/etcher/issues/734
Signed-off-by: Juan Cruz Viotti <jviotti@openmailbox.org>

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -6939,9 +6939,9 @@
       "dev": true
     },
     "unbzip2-stream": {
-      "version": "1.0.10",
-      "from": "unbzip2-stream@>=1.0.10 <2.0.0",
-      "resolved": "https://registry.npmjs.org/unbzip2-stream/-/unbzip2-stream-1.0.10.tgz"
+      "version": "1.0.11",
+      "from": "unbzip2-stream@1.0.11",
```
Comment (lurch) [inline npm-shrinkwrap.json]: Just another curiosity-question: Given the small change in `package.json`, why has the `"from":` line changed like this?

Comment (jviotti) [inline npm-shrinkwrap.json]: I believe its caused by my npm upgrade, which triggered this change given we've updated this dependency from it.

Comment (lurch) [inline npm-shrinkwrap.json]: Is it possible to "regenerate" `npm-shrinkwrap.json` using this upgraded npm format, to keep future commits "cleaner", or am I just worrying about nothing again?

Comment (jviotti) [inline npm-shrinkwrap.json]: `from` only seems to change when you explicitly modify a dependency. Looks like we would have to uninstall/install every single dependency here to make such a change :(

Comment (lurch) [inline npm-shrinkwrap.json]: Ah well, fair enough then ;-)

**Pull request merged**